api/src/main/java/org/openmrs/module/eptsreports/EptsReportsActivator.java

```diff
@@ -38,6 +39,9 @@ public void contextRefreshed() {
   @Override
   public void willRefreshContext() {
     log.debug("Refreshing EPTS Reports Module");
+    EptsGlobalPropertyService eptsGlobalPropertyService =
+        Context.getRegisteredComponents(EptsGlobalPropertyService.class).get(0);
```

Comment:
I'm getting this error:

    WARN - ModuleUtil.refreshApplicationContext(814) |2020-07-06 12:42:00,323| Unable to call willRefreshContext() method in the module's activator
    java.lang.IndexOutOfBoundsException: Index: 0, Size: 0
        at java.util.ArrayList.rangeCheck(ArrayList.java:635)
        at java.util.ArrayList.get(ArrayList.java:411)
        at org.openmrs.module.eptsreports.EptsReportsActivator.willRefreshContext(EptsReportsActivator.java:43)
        at org.openmrs.module.ModuleUtil.refreshApplicationContext(ModuleUtil.java:810)
        at org.openmrs.module.web.WebModuleUtil.refreshWAC(WebModuleUtil.java:962)
        at org.openmrs.web.Listener.performWebStartOfModules(Listener.java:627)
        at org.openmrs.web.Listener.performWebStartOfModules(Listener.java:606)
        at org.openmrs.web.Listener.startOpenmrs(Listener.java:249)
        at org.openmrs.web.WebDaemon$1.run(WebDaemon.java:42)

It seems DI is not working properly. 
Try using `moduleApplicationContext.xml` or a different annotation for `EptsGlobalPropertyService`.
OpenMRS does weird stuff with DI.